redis-test/LICENSE

```diff
@@ -0,0 +1,31 @@
+Copyright (c) 2022 by redis-rs contributors
```

Comment:
I usually do this by adding a symlink to the parent-dir `LICENSE`, which avoids the need to keeping this in sync.